src/ui/icons/leftArrow.tsx

```diff
@@ -1,6 +1,6 @@
 import React from "react";
 
-const Flechegauche = () => {
+const LeftArrow = () => {
```

Comment:
this element should take in props and then used to change the svg value if wanted. @youssef-night will explain this to u